src/lib.rs

```diff
@@ -107,7 +107,7 @@
 //!
 //! # Migrating from 0.5 to 0.6
 //!
-//! Please read [the migration guide](https://github.com/jamwaffles/embedded-graphics/blob/master/embedded-graphics/MIGRATING-0.5-0.6.md).
+//! Please read [the migration guide](https://github.com/embedded-graphics/embedded-graphics/blob/master/embedded-graphics/MIGRATING-0.5-0.6.md).
```

Comment:
```suggestion
//! Please read [the migration guide](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.5-0.6.md).
```
Or should we move the migration guides to the `docs` folder?

Reply:
I think leaving it in the root makes it more accessible (and also requires fewer changes and I'm lazy) but I'm not opposed to moving it to `doc/` later - we do link the migration guides from the readme anyway.